feat: basic state (#69)
keephq/keep: docs/docs/throttles/what-is-throttle.md

---
sidebar_label: What is a Throttle?
sidebar_position: 1
---

# ❓ What is a Throttle

> The purpose of throttling is to prevent any action to be executed too many times, thus generating too many alerts.

Reviewer: ```suggestion
The purpose of throttling is to prevent any action from being triggered too many times, thus generating too many alerts.
```
